### PR TITLE
{math}[foss/2021b] Clp v1.17.7, CoinUtils v2.11.6, Osi v0.108.7

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
node3138.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/355108ec4ea53d049f7f6948e134b1bb for a full test report.

**Comment by jfgrimm** [comment]
@boegel I've merged MUMPS from #15128, can you remove it from this PR?

**Comment by boegel** [comment]
> @boegel I've merged MUMPS from #15128, can you remove it from this PR?

@jfgrimm done!

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node141.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/aebbc7c76abf93fc30906bf9d6ded590 for a full test report.

File: easybuild/easyconfigs/c/Clp/Clp-1.17.7-foss-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    ('Osi', '0.108.7'),
+    ('bzip2', '1.0.8'),
+    ('zlib', '1.2.11'),
+]
+
+# Use BLAS/LAPACK from OpenBLAS
+configopts = '--with-blas="$LIBBLAS" --with-lapack="$LIBLAPACK" '
+
+# Use METIS AND MUMPS from EB
+# --with-metis-lib is ignored
```
Comment (jfgrimm) [inline easybuild/easyconfigs/c/Clp/Clp-1.17.7-foss-2021b.eb:37]: is this correct? We have `--with-metis-lib` in `configopts`, and it does seem to be picked up

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @boegel!

**Pull request merged**